src/patterns/utils/test/a11y.js

```diff
@@ -21,6 +21,7 @@ function launchChrome() {
     .then((newChrome) => {
       chrome = newChrome;
       chromeHost = chrome.host;
+      // Todo: make this dynamic
       serverUrl = "http://localhost:3333/";
```

Comment:
Fractal used to have this method we could use to handle this URL dynamically in our tests. Patternlab doesn't have one.

Reply:
There are not any good options that I came accross in the browsersync API to get the current running port.

I was able to find util called portscanner, but the best we can get there is to return the first active port. We would need some way to confirm that the node server running at the returned port is our Browsersync instance.

There are some HTTP methods we can call like `reload` that would give us a 200 response if the request is made to a browsersync server.  

something like

```
http.request(
          `http://${server.instance.utils.devIp()}:${port}/__browser_sync__?method=reload`,
          (res) => ... 
```

This could work, but we would have trouble knowing if this was our specific instance of multiple browsersync servers running.

tbh this magic number might be the best approach.